src/name_bazaar/ui/subs/offerings_subs.cljs

```diff
@@ -142,15 +142,13 @@
      (subscribe [:offering/ens-record offering-address])
      (subscribe [:offering/registrar-registration offering-address])])
   (fn [[{:keys [:offering/top-level-name?]} ens-record registrar-registration] [_ offering-address]]
-    (when (and (:ens.record/owner ens-record)
-               (if top-level-name?
-                 (:name-bazaar-registrar.registration/owner registrar-registration)
-                 true))
-      (= offering-address
-         (:ens.record/owner ens-record)
-         (if top-level-name?
-           (:name-bazaar-registrar.registration/owner registrar-registration)
-           offering-address)))))
+    (when
+      (if top-level-name?
```

Comment:
Can we just directly compare if the owner of the domain (or registration) is equal to the offering address?

Reply:
As it is the thing handles falsey values, returning nil. Note that it depends on subscriptions and is itself called in `subscribe`s, I think the nils represent missing / not yet present values and are desired in the [subscription workflows](https://purelyfunctional.tv/guide/re-frame-building-blocks/#subscriptions).

Reply:
Yeah, I understood it, but it took me some time. I think we can make it more readable.

Can you rewrite that using `let`? You can make the `not nil` check explicit and compare with the variable created in the let statement:
```clojure
    (let [node-owner (if top-level-name?
                       (:name-bazaar-registrar.registration/owner registrar-registration)
                       (:ens.record/owner ens-record))]
      (and node-owner (= offering-address node-owner)))))
```

Reply:
Right that's nicer.